cmd/cosign/cli/clean.go

```diff
@@ -46,6 +47,14 @@ func Clean() *cobra.Command {
 }
 
 func CleanCmd(ctx context.Context, regOpts options.RegistryOptions, cleanType, imageRef string) error {
+	ok, err := cosign.ConfirmPrompt("WARNING: this will remove all signatures from the image")
```

Comment:
Two quick things: this actually deletes SBOMs and Attestations too.

And we should detect if there's an interactive session before prompting, and allow a -f flag to force during a script.